homeassistant/components/conversation/default_agent.py

```diff
@@ -699,14 +717,14 @@ async def _match_triggers(self, sentence: str) -> ConversationResult | None:
             "'%s' matched %s trigger(s): %s",
             sentence,
             len(matched_triggers),
-            matched_triggers,
+            list(matched_triggers.keys()),
```

Comment:
Keys is the default 
```suggestion
            list(matched_triggers),
```

Reply:
fixed in [25e2b3a](https://github.com/home-assistant/core/pull/97236/commits/25e2b3a85223a7a4f133b283c482572ee462fa85)